definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-pet-amend-nonprod.json

```diff
@@ -11,7 +11,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "AwaitingAmendCase",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "R"
+    "CRUD": "CRU"
```

Comment:
why?

Reply:
Yep, noticed recently that the authorisations on states define whether a user can or cannot run an event on a case in a particular state.
For instance here, "R" meant that the solicitor could see cases in "AwaitingAmendCase" but not to run any event on it.
That's why I had to change to "CRU".